content/doc/book/pipeline/cps-method-mismatches.adoc

```diff
@@ -0,0 +1,250 @@
+---
+layout: section
+title: Pipeline CPS Method Mismatches
+---
+ifdef::backend-html5[]
+:notitle:
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+:hide-uri-scheme:
+:toc:
+endif::[]
+
+= Pipeline CPS Method Mismatches
+
+Jenkins Pipeline uses a library called Groovy CPS to run Pipeline
+scripts. While Pipeline uses the Groovy parser and compiler, unlike a
+regular Groovy environment it runs most of the program inside a special
+interpreter. This uses a continuation-passing style (CPS) transform to
+turn your code into a version that can save its current state to disk (a
+file called `+program.dat+`  inside your build directory) and continue
+running even after Jenkins has restarted. (You can get some more
+technical background on
+the https://github.com/jenkinsci/workflow-cps-plugin/blob/master/README.md[plugin page]
```

Comment:
Since the plugin has switched to deliver its documentation as code, we could link to the plugin page rather than the GitHub README page.
```suggestion
the plugin:workflow-cps[Pipeline: Groovy plugin page]
```

Reply:
Done in bb2cfe7.